lib/build.test.js

```diff
@@ -1,5 +1,5 @@
-const { expect, log, fixturePath } = require('./test');
-const { Builder } = require('../lib/build');
+const { expect, log, fixturePath } = require('../test/test');
```

Comment:
Nit: rename test.js to index.js so that test utilities can be imported all at once.

Reply:
Good call, took care of that in https://github.com/particle-iot/device-os-test-runner/pull/14/commits/ca6dde435f212d8d3a54ba939a81062d6395157f

> I'm not sure I get the idea of having stubs for tests

It serves the developer need of wanting to know what code is risky to change because it doesn't have any test coverage. In this case, it highlights that it is indeed risky to change most of the code in this project because there is only 20% test coverage.

Additionally, I like to add stubs like this because I want to lower the barrier to writing tests next time some aspect of the system needs to change. Having files already created for each piece of application functionality makes it obvious where test code should go when it's needed.

I'll merge this now, cut a release, and announce in #eng-device-os. #14 
Thanks for the review.